xbmc/filesystem/CurlFile.cpp

```diff
@@ -837,6 +841,11 @@ void CCurlFile::ParseAndCorrectUrl(CURL &url2)
         {
           SetCustomRequest(value);
         }
+        if (name == "verifypeer")
```

Comment:
@rbuehlma try with `else if`

I'll fix it.

Reply:
Yap, I can confirm this does fix my issue :) Thank you very much.